### PR TITLE
Cleanup metrics for local cache

### PR DESCRIPTION
**Pull request by apc999**
- Improve metrics counting for put/get/delete operations in LocalCacheManager
- separate counting different possible failure reasons with different counters

### CODE REVIEW

File: core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java [hunk 1]
```diff
@@ -238,7 +241,10 @@ public boolean put(PageId pageId, byte[] page) {
     try {
       mAsyncCacheExecutor.submit(() -> {
         try {
-          putInternal(pageId, page);
+          boolean ok = putInternal(pageId, page);
+          if (!ok){
```
Comment (calvinjia) [inline core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java:245]: white space missing

Comment (apc999) [reply core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java]: fixed

File: core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java [hunk 2]
```diff
@@ -238,7 +241,10 @@ public boolean put(PageId pageId, byte[] page) {
     try {
       mAsyncCacheExecutor.submit(() -> {
         try {
-          putInternal(pageId, page);
+          boolean ok = putInternal(pageId, page);
+          if (!ok) {
+            Metrics.PUT_ERRORS.inc();
```
Comment (calvinjia) [inline core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java:246]: I would prefer to do all the metric incrementing in the `putInternal` method.

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to master please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Pull request merged**